Examples/column-swiftui-example/column-ios/ContentView.swift

```diff
@@ -35,10 +33,11 @@ struct ContentView: View {
                         .border(Color(.white), width: 5)
                 }.sheet(isPresented: $activeWebView, content: {
                     // open the Column Tax SDK
-                    ColumnTaxView(
-                        userUrlRequest: URLRequest(url: URL(string: getUrlText())!),
+                    ColumnTaxFile(
+                        userUrl: URL(string: getUrlText())!,
                         isPresented: self.$activeWebView,
-                        onClose: self.handleClose
+                        handleClose: self.handleClose,
+                        environment: .sandbox
```

Comment:
Same comment here on the environment: https://github.com/column-tax/column-tax-react-native-sdk/pull/2#discussion_r1623092019